src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -144,7 +158,9 @@ public function __construct(
         SessionManagerInterface $sessionManager,
         EventDispatcherInterface $eventDispatcher,
         ResourceLocatorStrategyPoolInterface $resourceLocatorStrategyPool,
-        NamespaceRegistry $namespaceRegistry
+        NamespaceRegistry $namespaceRegistry,
+        AccessControlManagerInterface $accessControlManager,
```

Comment:
This can be null if website has no security.

Reply:
That's not true, this service is from our `SecurityBundle`, which is always registered because we need our user entities.